app/forms/registration_form.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require 'byebug'
+
+class RegistrationForm < ApplicationForm
+  def save(create_params)
+    return unless validate_client(create_params[:client_id])
```

Comment:
Because in the controller we have `if registration_form.save(create_params)`, therefore in this FormObject we should return true/false for consistency check from the controller. 

Please check other return below as well 

```suggestion
    return false unless validate_client(create_params[:client_id])
```